packages/reverse-proxy-service/src/restricted-spas/resolver.ts

```diff
@@ -12,8 +13,9 @@ function resolver ( req: Request, res: Response, next: NextFunction ): void {
     toProxy: true,
     ignorePath: false,
     headers: {
-      'X-OP-Authenticated': 'true',
+      'X-OP-Authenticated': req.oidc.isAuthenticated() ? 'true' : 'false',
```

Comment:
How about String(req.oidc.isAuthenticated())

Reply:
It's more readable this way imo.